examples/chip-tool-darwin/commands/common/CHIPCommandBridge.mm

```diff
@@ -72,16 +72,8 @@
     return CHIP_NO_ERROR;
 }
 
-#if !CONFIG_USE_SEPARATE_EVENTLOOP
-static void OnResponseTimeout(chip::System::Layer *, void * appState)
-{
-    (reinterpret_cast<CHIPCommandBridge *>(appState))->SetCommandExitStatus(CHIP_ERROR_TIMEOUT);
-}
-#endif // !CONFIG_USE_SEPARATE_EVENTLOOP
-
 CHIP_ERROR CHIPCommandBridge::StartWaiting(chip::System::Clock::Timeout duration)
 {
-#if CONFIG_USE_SEPARATE_EVENTLOOP
     chip::DeviceLayer::PlatformMgr().StartEventLoopTask();
```

Comment:
Followup: we shouldn't even call `StartEventLoopTask` here, since controller bringup did that already....

Reply:
Ok. Created an issue https://github.com/project-chip/connectedhomeip/issues/17366.